spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/GcpContextAutoConfiguration.java

```diff
@@ -76,4 +79,19 @@ public GcpProjectIdProvider gcpProjectIdProvider() {
 				? () -> this.gcpProperties.getProjectId()
 				: new DefaultGcpProjectIdProvider();
 	}
+
+	/**
+	 * Google Client Library APIs (i.e., non-Veneer/google-cloud-java) require
```

Comment:
When we go to the `HttpCredentialsAdapter`, we won't need this bean definition at all. Therefore "Veneer" will go.